`project_service` update to read/acquire a service on create

c2thorn opened the pull request:
Instead of trying to enable the API every time on Create, I added a read to check if the API was already enabled first.
<!-- AUTOCHANGELOG for Downstream PRs.

EXTERNAL CONTRIBUTORS: Ignore please - your reviewer will handle.

INTERNAL CONTRIBUTORS AND REVIEWERS: See .ci/RELEASE_NOTES_GUIDE.md
for writing good release notes.

NO CHANGELOG NOTE: Please add "changelog: no-release-note" label to this PR.

Otherwise, fill the template out (replace the heading).
You can add more release notes if you want more than one CHANGELOG entry for
this PR, but make sure not to indent notes and to leave newlines between
code blocks for Markdown's sake.

For Terraform PRs, we use the following "release-note:" headings
    - release-note:enhancement
    - release-note:bug
    - release-note:note
    - release-note:new-resource
    - release-note:new-datasource
    - release-note:deprecation
    - release-note:breaking-change
    - release-note:none
-->

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
`google_project_service` will no longer attempt to enable a service that is already enabled.
```

modular-magician commented:
Hi!  I'm the modular magician.  Your PR generated some diffs in downstreams - here they are.
# Diff report:


Terraform GA: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-3182-old..auto-pr-3182) ( 1 file changed, 18 insertions(+), 4 deletions(-))
Terraform Beta: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3182-old..auto-pr-3182) ( 1 file changed, 18 insertions(+), 4 deletions(-))

c2thorn commented:
Updated! The docs already had a little blurb that was relevant, so I just modified it to say "verify" instead of "re-enable" an already enabled service.

modular-magician commented:
Hi!  I'm the modular magician.  Your PR generated some diffs in downstreams - here they are.
# Diff report:


Terraform GA: [Diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-3182-old..auto-pr-3182) ( 2 files changed, 19 insertions(+), 5 deletions(-))
Terraform Beta: [Diff](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3182-old..auto-pr-3182) ( 2 files changed, 19 insertions(+), 5 deletions(-))
